refarch-frontend/src/plugins/vuetify.ts

```diff
@@ -0,0 +1,20 @@
+import "@mdi/font/css/materialdesignicons.css";
+import "vuetify/styles";
+
+import { createVuetify } from "vuetify";
+
+export default createVuetify({
+  theme: {
+    themes: {
+      light: {
+        colors: {
+          primary: "#333333",
+          secondary: "#FFCC00",
+          accent: "#7BA4D9",
+          success: "#69BE28",
+          error: "#FF0000",
+        },
+      },
```

Comment:
We could think about separating theme definition into separate file and then import here. Might make the whole theming stuff more clear especially when we add things like dark mode, high contrast mode etc.

Reply:
not sure why we would do this. If you want to change the vuetify theme, you need to know where you have to change it and that should be in the vuetify documentation. Moving thing away from it doesnt sound like a good idea.